packages/engine/rustfmt.toml

```diff
@@ -13,7 +13,7 @@ wrap_comments = true # Default: false
 
 # Parameters
 single_line_if_else_max_width = 50 # Default: 50
-comment_width = 100 # Defaut: 80
+comment_width = 80 # Defaut: 80
```

Comment:
We agreed on **not** doing this, can you revert this commit please?

Reply:
Yeah, I'm still changing the PR